src/configs/kubernetes.rs

```diff
@@ -32,6 +33,20 @@ impl<'a> Default for KubernetesConfig<'a> {
             detect_extensions: vec![],
             detect_files: vec![],
             detect_folders: vec![],
+            environments: vec![],
         }
     }
 }
+
+#[derive(Clone, Deserialize, Serialize, Default)]
```

Comment:
I am not as familiar with schemars/serde as I'd like to be, so I'd like a quick check from another maintainer on this part to make sure the macros on this struct look correct.

Reply:
The macros look correct to me.